contentcuration/contentcuration/serializers.py

```diff
@@ -602,7 +624,20 @@ class Meta:
             'original_channel', 'original_source_node_id', 'source_node_id', 'content_id', 'original_channel_id',
             'source_channel_id', 'source_id', 'source_domain', 'thumbnail_encoding', 'publishing',
             'children', 'parent', 'tags', 'created', 'modified', 'published', 'extra_fields', 'assessment_items',
-            'files', 'valid', 'metadata', 'tree_id', 'freeze_authoring_data', 'role_visibility')
+            'files', 'valid', 'metadata', 'tree_id', 'freeze_authoring_data', 'role_visibility', 'provider', 'aggregator')
+
+
+class TokenSerializer(serializers.ModelSerializer):
+    display_token = serializers.SerializerMethodField('generate_token')
+
+    def generate_token(self, token):
+        return "{}-{}".format(token.token[:5], token.token[5:])
```

Comment:
How longs are tokens usually? If we want to break them up, we may need something more sophisticated if they get longer.

Reply:
They are only going to be 10 characters as that's how proquint handles the token logic